likecoin/public/iscn-badge.php

```diff
@@ -31,17 +31,34 @@
  * @param WP_Post| $post Post object.
  */
 function likecoin_add_iscn_badge( $post ) {
-	$post_id               = $post->ID;
-	$iscn_info             = get_post_meta( $post_id, LC_ISCN_DEV_INFO, true );
-	$iscn_hash             = isset( $iscn_info['iscn_hash'] ) ? $iscn_info['iscn_hash'] : null;
-	$option                = get_option( LC_PUBLISH_OPTION_NAME );
-	$should_add_iscn_badge = checked( $option[ LC_OPTION_SITE_ADD_ISCN_BADGE ], 1, false );
+	$post_id   = $post->ID;
+	$iscn_info = get_post_meta( $post_id, LC_ISCN_DEV_INFO, true );
+	$iscn_hash = isset( $iscn_info['iscn_hash'] ) ? $iscn_info['iscn_hash'] : null;
+	$option    = get_option( LC_PUBLISH_OPTION_NAME );
 
-	if ( strlen( $iscn_hash ) > 0 && $should_add_iscn_badge ) {
+	$is_dark_badge   = 0; // default is showing white badge.
+	$show_iscn_badge = 'yes';
```

Comment:
Just use a boolean True/False....

Reply:
true...lol

Reply:
not native enough yet for programming language xD